Legion.BackEnd/database/migrations/2018_07_17_021708_create_announcements_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateAnnouncementsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('announcements', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('title');
+            // Will be able to store up to 64KiB a field with play TEXT field. Should be good enough.
+            // Can go up to 4 GiB if needed with LONGTEXT...
```

Comment:
I don't think we'll ever exceed 64KiB lol

Reply:
The one below it is like 255 chars so, meh.

Reply:
That's fine.  Do you know if it uses up 64KiB per record, or if it's smart and only uses what it needs?

Reply:
It uses what it needs.

Reply:
https://stackoverflow.com/questions/37518598/space-occupied-by-text-type-column-in-mysql